src/components/Strategies/StrategyList.tsx

```diff
@@ -5,13 +5,25 @@ import { PlusOutlined } from '@ant-design/icons';
 import styled from 'styled-components';
 
 import { IStrategy } from './../../types';
+import { useAuth } from '../../hooks/useAuth';
 import { strategies } from './strategyData';
 
 const { Meta } = Card;
 const { Text } = Typography;
+const strategyPathKey = 'strategyPath';
 
 function StrategyList() {
   const history = useHistory();
+  const { user } = useAuth();
+
+  const checkUserLoggedIn = (strategyUrl: string) => {
```

Comment:
Very bad naming again. Ask yourself - what are you doing here? Is it just checking? Checking implies only validation and nothing else. This function does something different so name it accordingly